packages/SwingSet/bin/extract-transcript-from-kerneldb.js

```diff
@@ -0,0 +1,109 @@
+import lmdb from 'node-lmdb';
+import process from 'process';
+import fs from 'fs';
+
+const argv = process.argv.splice(2);
+const dirPath = argv[0];
+const vatName = argv[1];
+
+if (!dirPath) {
+  console.log('extract-transcript-from-kerneldb DBDIR : list all vats in DB');
+  console.log(
+    'extract-transcript-from-kerneldb DBDIR VATID : extract transcript',
```

Comment:
Since (it looks like) you allow the vat to be specified by either a vatID or a vat name, usage text should say this.